Automattic/woocommerce-payments
# Add the default currency and exchange rate to new orders' meta

Fixes #1777 

#### Changes proposed in this Pull Request

As part of the analytics effort, we'll have to display the order prices converted to the store's base currency in the reports. To do so, this PR introduces two meta fields to the order: `_wcpay_multi_currency_order_exchange_rate` and `_wcpay_multi_currency_order_default_currency`.

The customer total and currency do not have to be stored in the meta because WooCommerce saves them in the order data already. Are the two metas above enough for our use case or should we store something else?









#### Testing instructions





* Create an order using a currency that needs to be converted
* Check that the order's metadata for the `_wcpay_multi_currency_order_exchange_rate` and `_wcpay_multi_currency_order_default_currency` keys are correct
* Repeat the test using the checkout block

-------------------
